src/binder/assemblybinder.cpp

```diff
@@ -364,7 +364,7 @@ namespace BINDER_SPACE
     {
         HRESULT hr = S_OK;
  
-       if (!BINDER_SPACE::fAssemblyBinderInitialized)
+        if (!BINDER_SPACE::fAssemblyBinderInitialized)
```

Comment:
You can delete `fAssemblyBinderInitialized`. It is not needed.

Reply:
It prevents issues calling Startup twice (and asserts that startup was called before anything else).
I guess I don't mind removing it, but it does have at least some value.

Reply:
We have many initialize methods in VM. Majority of them do not bother with this and just assume that EEStartup that calls them does the right thing (tm).